11.gün/index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Task List</title>
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.2.0/css/all.css">
+    <link rel="stylesheet" href="../11.gün/style.css">
```

Comment:
.. ile geri dönüp tekrar 11. güne girmene gerek yok. firek style.css de olur. 
```suggestion
    <link rel="stylesheet" href="style.css">
```

Reply:
Bununla ilgili daha genel bir kullanim söyle. Bir css klasorü acip style.css dosyasini oraya koymak. oradan cagirmak.

Reply:
günleri ayrı ayrı klasörlediğim için mecbur her klasöre css açıyorum ve ordan çağırıyorum.

Reply:
Tamam yine de öyle cagirmana gerek yok, href="style.css" sekline cagirabilirsin. 
Dosya yolu belirtirken baslangic noktan icinde bulundugun dosyanin oldugu yer. Yani sen zaten 11.Gün klasörü icindesin. ".." koyarak önce klasör disina cikip sonra geri ayni klasöre giriyorsun.

Reply:
örnek klasör yapisi icin dev_2 branchina bak

Reply:
anladım.